src/rulesets/Random/random.jl

```diff
@@ -15,51 +15,27 @@ end
 @non_differentiable rand()
 @non_differentiable rand(::AbstractRNG)
 @non_differentiable rand(::AbstractRNG, ::Random.Sampler)
-@non_differentiable rand(::AbstractRNG, ::Integer)
-@non_differentiable rand(::AbstractRNG, ::Integer, ::Integer)
-@non_differentiable rand(::AbstractRNG, ::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::AbstractRNG, ::Integer, ::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::AbstractRNG, ::Integer, ::Integer, ::Integer, ::Integer, ::Integer)
+@non_differentiable rand(::AbstractRNG, ::Integer...)
 @non_differentiable rand(::Type{<:Real})
 @non_differentiable rand(::Type{<:Real}, ::Tuple)
-@non_differentiable rand(::Type{<:Real}, ::Integer)
-@non_differentiable rand(::Type{<:Real}, ::Integer, ::Integer)
-@non_differentiable rand(::Type{<:Real}, ::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::Type{<:Real}, ::Integer, ::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::Type{<:Real}, ::Integer, ::Integer, ::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::Integer)
-@non_differentiable rand(::Integer, ::Integer)
-@non_differentiable rand(::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::Integer, ::Integer, ::Integer, ::Integer)
-@non_differentiable rand(::Integer, ::Integer, ::Integer, ::Integer, ::Integer)
-
+@non_differentiable rand(::Type{<:Real}, ::Integer...)
+@non_differentiable rand(::Integer...)
 
 # There are many different 1-3 arg methods, but not varargs
```

Comment:
I'm unsure about what this comment means, so I didn't replace it by `rand!(::Any...)`

Reply:
Now I see, thank you!